doc/api/os.md

```diff
@@ -451,6 +451,8 @@ system. This differs from the result of `os.homedir()`, which queries several
 environment variables for the home directory before falling back to the
 operating system response.
 
+Throws a `SystemError` if user has no `username` or `homedir`.
```

Comment:
Would be useful if `SystemError` linked to https://nodejs.org/api/errors.html#errors_system_errors to make it quick for users to look up what it is.

Reply:
Added.